.github/scripts/revert-commit-draft-release.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+echo "Running revert last commit script"
+echo $1
+echo "Reverting commit:"
+echo $(git rev-parse --verify HEAD)
+echo "Git configurations :"
+git config --global user.email "team_interfaces+github@twilio.com"
+git config --global user.name "twilio-dx"
+git revert $(git rev-parse --verify HEAD)
+git push origin refs/heads/$1:refs/heads/$1
+echo "Reverting commit:"
+echo $(git rev-parse --verify HEAD)
+git revert $(git rev-parse --verify HEAD)
+git push origin refs/heads/$1:refs/heads/$1
```

Comment:
Are we not reverting the last two commits? 
1. semantic bot release commit
2. open api definitions commit

Reply:
Yes, we are. There are 2 revert statements at line 9 and 13